site/content/entry/citra-now-has-an-installer.md

```diff
@@ -0,0 +1,65 @@
++++
+date = "2017-10-01T00:00:00-05:00"
+title = "Citra now has an installer!"
+tags = [ "feature-update" ]
+author = "thekoopakingdom"
+forum = 0000
++++
+
+Emulators grow quickly. Last month alone, there were [39 pull requests](https://github.com/citra-emu/citra/pulls?utf8=%E2%9C%93&q=is%3Amerged%20merged%3A2017-09-01..2017-09-30%20)
+merged to Citra. For this reason, it's important to be able to manage the many different
+versions of an emulator. Until now however, Citra has had no installer or updater
+for the [nightly builds](https://github.com/citra-emu/citra-nightly/releases). Although
+the now obsolete [Bleeding Edge builds](https://github.com/citra-emu/citra-bleeding-edge/releases)
+did have both of these things, it was limited to Windows, and the installer framework
+had some issues that restricted what we could do with it.
+
+Thanks to the work by [j-selby](https://github.com/j-selby) and [jroweboy](https://github.com/jroweboy),
+we now have a new installer and updater for both the [Nightly builds](https://github.com/citra-emu/citra-nightly/releases)
+and the [Canary builds](https://github.com/citra-emu/citra-canary/releases)!
+
+More info on the Canary version of Citra will be in the **September 2017 Progress
+Report**.
```

Comment:
October progress report?

Reply:
I believe it still is called the September Progress Report despite being delayed.